### PR TITLE
[#2653] Support Browser Dispose Timeouts in Hovers

### PR DESCRIPTION
**Pull request by cdietrich**
[#2653] Support Browser Dispose Timeouts in Hovers

### CODE REVIEW

File: org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/editor/hover/html/DefaultEObjectHoverProvider.java [hunk 1]
```diff
@@ -370,6 +370,7 @@ public IInformationControl doCreateInformationControl(Shell parent) {
 				ToolBarManager tbm = new ToolBarManager(SWT.FLAT);
 				String font = "org.eclipse.jdt.ui.javadocfont"; // FIXME: mPreferenceConstants.APPEARANCE_JAVADOC_FONT;
 				IXtextBrowserInformationControl control = new XtextBrowserInformationControl(parent, font, tbm);
+				control.setDisposeTimeout(10000);
```
Comment (cdietrich) [inline org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/editor/hover/html/DefaultEObjectHoverProvider.java:373]: will this call the default method in the interface or the one in the baseclass

File: org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/editor/hover/html/XtextBrowserInformationControlAdapter.java [hunk 2]
```diff
@@ -138,4 +146,19 @@ public void addLocationListener(LocationListener createLocationListener) {
 	public void addInputChangeListener(IInputChangedListener inputChangeListener) {
 		control.addInputChangeListener(inputChangeListener);
 	}
+
+	@Override
+	public void setDisposeTimeout(int disposeTimeout) {
```
Comment (cdietrich) [inline org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/editor/hover/html/XtextBrowserInformationControlAdapter.java:151]: @szarnekow any idea how to test this at all?

**Comment by LorenzoBettini** [comment]
@cdietrich would it be OK to cancel all the previous GitHub Actions jobs related to this? We can leave the last one running. Otherwise, the queue is too full.

**Comment by cdietrich** [comment]
@LorenzoBettini  yes i am looking into jenkins results only anway

**Review by szarnekow — APPROVED** [review]
I don't know how to test this automatically, either.

XbaseInformationControl scares me.

**Pull request merged**